imagersite/imager_images/tests.py

```diff
@@ -70,7 +70,6 @@ def setUp(self):
             album = AlbumFactory.create(cover=cover, user=user.profile)
             album.save()
 
-    @classmethod
     def tearDown(self):
         Album.objects.all().delete()
         User.objects.all().delete()
```

Comment:
This should not be required.  The objects you create should be removed at the end of each test anyway.